yellowpages.lic

```diff
@@ -0,0 +1,56 @@
+#Load base contacts YAML
+Contacts = YAML.load_file('./scripts/data/base-contacts.yaml')
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+#puts "---Base Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?('./scripts/profiles/'+checkname+'-contacts.yaml')

```

Comment:
Our convention has been to use string interpolation for cases like this (e.g. `"./scripts/profiles/#{checkname}-contacts.yaml"`).